docs/quality-assurance-test-cases.md

```diff
@@ -174,6 +174,31 @@ This document contains some examples of test cases for each feature implemented
 | Logged in user tries to complete a task from a request which is not in the ACCEPTED state (as a mentor or as a mentee) | Fail |
 
 
+### Tasks
+
+#### Create
+
+**Service:** POST /mentorship_relation/{relation_id}/task
+
+|  Test Case                                                                                | Outcome |
+| ----------------------------------------------------------------------------------------- |-------- |
+| Create a task for a relation, in the accepted state, between logged user and another user | Success |
+| Creating a task without a description (either empty or not in the request body at all)    | Fail    |
+| Create a task when a logged user is not involved in the relation                          | Fail    |
+| Create a task if relation state is different than accepted                                | Fail    |
+
```

Comment:
this is already present on the `develop` branch of the project. Is your fork up to date with this repository @khushishikhu ?

Reply:
I see this is duplicated in this file, the same table is present on line 231, could you please remove the duplicate from below this line? So you only have this page here?

Reply:
yes sure I will remove the duplicate table from that line.Thank you!